File: inflate/tests/langs/kotlin/Box.kt
package sample

class Box {
    private val size = 1

    fun helper(value: Int): Int {
        return value + size
    }

    fun greet(value: Int): Int {
        return helper(value) * 2
    }

    fun runner(): Int {
        return greet(3)
    }
}

File: inflate/tests/langs/objc/box.m
#import <Foundation/Foundation.h>

@interface Box : NSObject
@end

@implementation Box

- (int)helper:(int)value {
    return value + 1;
}

- (int)greet:(int)value {
    return [self helper:value] * 2;
}

- (int)runner {
    return [self greet:3];
}

@end
